### ISSUE TITLE
IDS-Checkbox

### ISSUE THREAD

**Issue by ajayponnapati** [infor-design/enterprise-wc#2260]
**Describe the bug**
when using reactive forms in angular with ids-checkbox the value of the checkbox is either "on" or "" but it should actually be boolean

**To Reproduce**

1. Go to [''](https://github.com/infor-design/enterprise-wc-examples/blob/main/angular-ids-wc/src/app/components/ids-reactive-forms/demos/example/example.component.html)
2. Click on  testCheckbox and make it checked
3. console.log its value and you see '' and when you uncheck it its value is "on"

**Expected behavior**
the value should be true/false based on checked/unchecked the checkbox

**Version**

- ids-enterprise: ^1.0.0-beta.23 and ^1.0.0

**Screenshots**

**Platform**
 - Infor Application/Team Name: [e.g. Infor XYZ, Infor ABC]
 - Device: (if applicable) [e.g. iPhone 6 or Samsung Galaxy S6]
 - OS Version: [e.g. Windows 10 or iOS 8]
 - Browser Name: [e.g. chrome, safari, stock browser]
 - Browser Version: [e.g. 22, 66.0.3359.181 (Official Build) (64-bit)]

**Additional context**

**Comment by tmcconechy** [comment]
This is correct behavior. Basically value is the form name/value and checked is probably what you want to look at:

See:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value
vs
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#additional_attributes

**Comment by tmcconechy** [comment]
Based on discussion can change this and not follow the spec.

**Comment by AndrewDeiakCZ** [comment]
Hi @tmcconechy ,  will you change it to boolean? 
Just wanted to confirm 😊

cc @ajayponnapati

**Comment by tmcconechy** [comment]
Yes i think it should be typed to `boolean` or `boolean | undefined | nill`
